gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Tokens.java

```diff
@@ -30,9 +30,21 @@ private Tokens() {}
     public static final String OPS_BYTECODE = "bytecode";
     public static final String OPS_EVAL = "eval";
     public static final String OPS_INVALID = "invalid";
+    public static final String OPS_CLOSE = "close";
+
+    /**
+     * @deprecated As of release 3.3.8, not directly replaced in the protocol as side-effect retrieval after
```

Comment:
(nitpick) I think these tokens should also be deprecated the same way in Gremlin.Net's [`Token` class](https://github.com/apache/tinkerpop/blob/master/gremlin-dotnet/src/Gremlin.Net/Driver/Tokens.cs) (and I don't know about gremlin-python and gremlin-javascript).

Reply:
ok - added the obsolete annotation for .NET. javascript doesn't have side-effect functionality so nothing to deprecate there. i thought about python but it meant adding the `deprecations` library (i think) and i didnt' want to add a dependency (maybe we should...i dunno - perhaps that's a separate decision for newer versions of gremlin-python). how does everything look now?

Reply:
Looks good to me in general, but it looks like you need to add a `using` directive in `Tokens.cs` which lets the build fail right now.